.gitbook/5-oracles/0-about-dep-oracle.md

```diff
@@ -0,0 +1,81 @@
+# About DEP Oracle
+
+This document provides a brief description of oracle's role in the `DEP` ecosystem and a conceptual introduction to why we use the `SGX` environment.
+There is also a brief description of oracle reward for active oracle participation in the `DEP` ecosystem.
+
+## Role of Oracle
+
+Oracle is a key component of `data validation` in the `DEP` ecosystem.
+
+### Data Validator
+
+Oracle is a validator that validates data as it transmits from provider to consumer.
+
+Since data encryption is required for data privacy, `Panacea`, a public blockchain, cannot act as a validator.
+Instead, the oracle decrypts and validates the provider's data and provides the result in the form of a `certificate`.
+Then, data consumer and `Panacea blockchain` can check the `certificate` to determine the validity of the data.
```

Comment:
We called blockchain a `Panacea` above, so let's just call it `Panacea` here too
```suggestion
Then, data consumer and `Panacea` can check the `certificate` to determine the validity of the data.
```